specification/cognitiveservices/data-plane/WebSearch/v1.0/WebSearch.json

```diff
@@ -0,0 +1,1235 @@
+            "in": "query",
+            "description": "The user's search query term. The term may not be empty. The term may contain Bing Advanced Operators. For example, to limit results to a specific domain, use the site: operator.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "responseFilter",
+            "x-ms-client-name": "ResponseFilter",
+            "in": "query",
+            "description": "A comma-delimited list of answers to include in the response. If you do not specify this parameter, the response includes all search answers for which there's relevant data. Possible filter values are Computation, Images, News, RelatedSearches, SpellSuggestions, TimeZone, Videos, Webpages. Although you may use this filter to get a single answer, you should instead use the answer-specific endpoint in order to get richer results. For example, to receive only images, send the request to one of the Image Search API endpoints. The RelatedSearches and SpellSuggestions answers do not support a separate endpoint like the Image Search API does (only the Web Search API returns them). To include answers that would otherwise be excluded because of ranking, see the promote query parameter.",
+            "required": false,
+            "type": "array",
+            "items": {
+              "type": "string",
+              "enum": [
+                "WebPages",
+                "Images",
+                "SpellSuggestions",
+                "News",
+                "RelatedSearches",
+                "Videos",
+                "WebPageGroupings",
+                "Computation",
+                "TimeZone"
+              ],
+              "x-ms-enum": {
+                "name": "AnswerType",
+                "modelAsString": true
+              }
+            },
+            "collectionFormat": "csv"
+          },
+          {
+            "name": "safeSearch",
+            "x-ms-client-name": "SafeSearch",
+            "in": "query",
+            "description": "A filter used to filter adult content. Off: Return webpages with adult text, images, or videos. Moderate: Return webpages with adult text, but not adult images or videos. Strict: Do not return webpages with adult text, images, or videos. The default is Moderate. If the request comes from a market that Bing's adult policy requires that safeSearch is set to Strict, Bing ignores the safeSearch value and uses Strict. If you use the site: query operator, there is the chance that the response may contain adult content regardless of what the safeSearch query parameter is set to. Use site: only if you are aware of the content on the site and your scenario supports the possibility of adult content.",
+            "required": false,
+            "type": "string",
+            "enum": [
+              "Off",
+              "Moderate",
+              "Strict"
+            ],
+            "x-ms-enum": {
+              "name": "SafeSearch",
```

Comment:
add `values` section to set individual descriptions for the values.

Reply:
Tracked